serverless/installing_serverless/serverless-install-config-options.adoc

```diff
@@ -0,0 +1,87 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-install-config-options"]
+= Advanced installation configuration options
+include::modules/common-attributes.adoc[]
+:context: serverless-install-config-options
+
+toc::[]
+
+This guide provides information for cluster administrators about advanced installation configuration options for Knative Serving and Knative Eventing.
+
+The YAML configuration will be shown by default when you are creating the Knative Serving or Knative Eventing custom resource definitions. However, you can access the form view by selecting *Edit Form* in the top right corner of the *Create Knative Serving* or *Create Knative Eventing* pages.
+
+[IMPORTANT]
+====
+Do not modify any YAML contained inside the `config` field. Some of the configuration values in this field are injected by the {ServerlessOperatorName}, and modifying them will cause your deployment to become unsupported.
+====
+
+== Knative Serving supported installation configuration options
+
+image::serving-form-view.png[Create Knative Serving form]
+
+=== Controller Custom Certs
+
+To enable tag-to-digest resolution, the Knative Serving controller requires access to the container registry.
+
+If your registry uses a self-signed certificate, you must enable tag-to-digest resolution by creating a ConfigMap or Secret.
+The {ServerlessOperatorName} will then automatically configure Knative Serving controller access to the registry.
```

Comment:
I wonder if this paragraph should come first. 
Why you might need to enable t-to-d resolution.
How to do it.